src/components/data-management/CellLevelUploadModal.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import {
+  Modal,
+  Button,
+  Typography,
+  Space,
+  Row,
+  Col,
+  Empty,
+  Divider,
+  List,
+} from 'antd';
+import { CheckCircleTwoTone, DeleteOutlined } from '@ant-design/icons';
+import Dropzone from 'react-dropzone';
+
+import handleError from 'utils/http/handleError';
+import endUserMessages from 'utils/endUserMessages';
+import readFileToString from 'utils/upload/readFileToString';
+
+const { Text, Title, Paragraph } = Typography;
+
+const CellLevelUploadModal = (props) => {
```

Comment:
It doesn't look the same way as in the mockups https://docs.google.com/document/d/1YM43BaBYMKjg-Q9GbgDk0AyNnTgNwpIVif89utePu50/edit#heading=h.wisbg6di27up